spec/models/post_mover_spec.rb

```diff
@@ -2463,5 +2463,24 @@ def create_topic_user(user, topic, opts = {})
         end
       end
     end
+
+    describe "with event trigger" do
+      fab!(:topic_1) { Fabricate(:topic) }
+      fab!(:topic_2) { Fabricate(:topic) }
+      fab!(:post_1) { Fabricate(:post, topic: topic_1) }
+      fab!(:post_2) { Fabricate(:post, topic: topic_1) }
+
+      it "uses first_post_moved trigger for first post" do
```

Comment:
I think this should be it triggers "post_moved" and "first_post_moved" event when moving a first post.